packages/php-wasm/compile/php/Dockerfile

```diff
@@ -59,9 +59,12 @@ RUN touch /root/.configure-flags && \
 # PHP <= 7.3 requires Bison 2.7
 # PHP >= 7.4 and Bison 3.0
 RUN if [[ "${PHP_VERSION:0:1}" -le "7" && "${PHP_VERSION:2:1}" -le "3" ]]; then \
-		mv /libs/bison2.7/usr/local/bison /usr/local/bison && \
-        ln -s /usr/local/bison/bin/bison /usr/bin/bison && \
-		ln -s /usr/local/bison/bin/yacc /usr/bin/yacc; \
+		wget http://launchpadlibrarian.net/140087283/libbison-dev_2.7.1.dfsg-1_amd64.deb; \
+		wget http://launchpadlibrarian.net/140087282/bison_2.7.1.dfsg-1_amd64.deb; \
+		apt install -y --allow-downgrades ./libbison-dev_2.7.1.dfsg-1_amd64.deb; \
+		apt install -y --allow-downgrades ./bison_2.7.1.dfsg-1_amd64.deb; \
+		apt-mark hold libbison-dev; \
+		apt-mark hold bison; \
```

Comment:
What is the context behind this change? Why is it needed? It uses `http` which isn't secure. What is `launchpadlibrarian.net` and why not use a ubuntu repository? Also, what about the bison2.7 library files we already have?

Reply:
@adamziel 
That's an official source
* https://launchpad.net/
* http://bazaar.launchpad.net/~launchpad-pqm/launchpad/devel/view/head:/lib/lp/services/librarian/doc/librarian.txt

Apt doesn't always use HTTPS, so I copied the URLs as-is. Upon inspection, that server has got a cert so I'm updating it to use HTTPS.
https://archive.is/cd7iu

I can go ahead and remove the extraneous bison files. I don't think we need to maintain a compilation step if we can get it off-the-shelf from the OS vendor. Let me know what you think of that idea.

Reply:
Aha, thank you for explaining. I'm concerned this `.deb` URL could change without warning and break the build process – what do you think? 

> Apt doesn't always use HTTPS

Right, but APT verifies the signature of the downloaded packages.

Reply:
https://github.com/WordPress/wordpress-playground/blob/eebb534607d1537f98f00b64717a70c1b3cb81e3/packages/php-wasm/compile/php/Dockerfile#L60-L70